source/GameData.cpp

```diff
@@ -901,15 +906,28 @@ void GameData::LoadSources(TaskQueue &queue)
 	sources.clear();
 	sources.push_back(Files::Resources());
 
-	vector<string> globalPlugins = Files::ListDirectories(Files::Resources() + "plugins/");
-	for(const string &path : globalPlugins)
+	vector<string> plugins = Files::ListDirectories(Files::Resources() + "plugins/");
```

Comment:
This change doesn't seem necessary? Just have Plugins::IsPlugin accept zip files as plugins.